### ISSUE TITLE
Cannot import Web API

### ISSUE THREAD

**Issue by SvenC56** [syndesisio/syndesis#8752]
## This is a...

<pre><code>
[ ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Documentation issue or request
</code></pre>

## Description

Hey,

I'm trying to import a ~500kb open api document. On `review` step I get an empty warning and I cannot go further. I also tried a small API with 2 operation which is working (and also has warnings). Is this related the size of the json document? I looked into the logs of the `server` and `meta` but I did not found any warnings related to my import.

Thank you.

![image](https://user-images.githubusercontent.com/14073259/85378115-670f2d80-b53a-11ea-9b27-a03ca68e02b9.png)

**Comment by kahboom** [comment]
Hi @SvenC56 , would you mind sharing the documents you used so I can try to reproduce it? It looks like there is a size limit imposed but I don't seem to get the same empty alert (or one at all actually, don't know what's worse)..

**Comment by SvenC56** [comment]
Hey @kahboom ,

I shared the document in my OneDrive. Can you please notify me when you downloaded the file so I can remove it then.
https://svencieslok-my.sharepoint.com/:u:/g/personal/mail_svencieslok_de/EQZM_oo_8TZGjGP8yj2mJWIB8-HMXuKl7s9QpVYIiHiorw?e=zpPPdO

password: syndesis

Thank you.

**Comment by kahboom** [comment]
@SvenC56 - Sorry for the delay, I was working on a few other things and was only able to check now, but there is an error. I'm guessing you removed it by now or the link expired?

<img width="1213" alt="Screenshot 2020-06-30 08 53 54" src="https://user-images.githubusercontent.com/3844502/86101022-8fa2a480-bab1-11ea-9304-e47ea97d5c70.png">

**Comment by SvenC56** [comment]
Hey @kahboom,

I created a new link. It's valid until the end of the week.

https://svencieslok-my.sharepoint.com/:u:/g/personal/mail_svencieslok_de/EQZM_oo_8TZGjGP8yj2mJWIBus-DMjVIUX4a2UlqN6rbxQ?e=asHcj8

Password: syndesis

**Comment by kahboom** [comment]
Thanks @SvenC56 , I was able to download it. So, regarding uploading the specification, I've tried yours (1 MB), another 1 MB, and a 4.5 MB specification on one of our clusters. They were all slow, but the only one that didn't upload was the 4 MB one, and you would be able to see this in the console:

`DOMException: Failed to execute 'setItem' on 'Storage': Setting the value of 'kxms9b' exceeded the quota.`

I wasn't able to reproduce the alert either. It's possible that the Syndesis server is hitting a resource limit, so try taking a look at the pod log for it if you haven't already. There should be some kind of exception.

By the way, are you able to see anything in the network tab of the browser console? There should be a request to `/info` when you press 'Next' on Step 1, with some payload in the response.

![Kapture 2020-06-30 at 13 51 04](https://user-images.githubusercontent.com/3844502/86128159-ea9bc200-bad8-11ea-9b7b-6cfac0e485a5.gif)
